.circleci/config.yml

```diff
@@ -107,27 +114,48 @@ jobs:
       - run: yarn build:webpack
 
   build-e2e:
+    working_directory: ~/vets-website
     docker:
       - image: cypress/base:10.15.3
     steps:
       - checkout
       - restore_cache:
           keys:
-            - yarn-packages-{{ checksum "yarn.lock" }}
+            - yarn-packages-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - restore_cache:
+          keys:
+            - node-v1-{{ .Branch }}-{{ checksum "package.json" }}
       - run:
-          command: yarn install --frozen-lockfile
+          name: Install Dependencies
+          command: yarn --frozen-lockfile --cache-folder ~/.cache/yarn
+      - run:
+          name: Verify Cypress is Installed
+          command: yarn cypress verify || yarn cypress install
       - save_cache:
-          key: yarn-packages-{{ checksum "yarn.lock" }}
+          key: yarn-packages-{{ .Branch }}-{{ checksum "yarn.lock" }}
           paths:
-            - ~/.cache/yarn
-      - run: mkdir -p ./test-results
-      - run: yarn install-repos
-      - run: yarn fetch-drupal-cache
-      - run: yarn build
+            - ~/.cache
+      - save_cache:
+          key: node-v1-{{ .Branch }}-{{ checksum "package.json" }}
+          paths:
+            - node_modules
+      - run:
+          name: Make Test Results Folder
+          command: mkdir -p ./test-results
```

Comment:
Is this something that should happen in `e2e-test` rather than creating it in build and persisting it?

Reply:
I was debating about this also. I think the main reason why I put this here is because I wanted the `build-e2e` job to handle creating everything that the `e2e-test` job needs so we do as little work as possible in the `e2e-test` job to get tests running. I'm open to moving it to the `e2e-test` though. Maybe it's more efficient to do this in `e2e-test` rather than needing to persist it.

Reply:
I think its creation belongs in the `e2e-test` job, since it's the job that actually uses the folder. `build-e2e` doesn't actually do anything with it, and it won't get used at all if the build fails before getting to the tests.

Reply:
That's a good point. I'll move this too `e2e-test`.